### PR TITLE
Feature/dynamic files

### PR DESCRIPTION
**Pull request by flo7842**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,9 +30,11 @@ For Windows operating system, your icon extension should be (.ico).
 * For Windows to win32 folder, icon size should be at least 256x256.
 * For Linux to linux folder, recommended sizes: 16, 32, 48, 64, 128, 256 (or just 512).
 
+To change the default image with your own image, go to the config_electron_viewer.json file by passing the "img_default" option to false and then add your own image to the public/img folder.
```
Comment (greg7859) [inline README.md]: This is changed, we have to set the path to the new image

File: main.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+            nodeIntegration: true,
+            contextIsolation: false,
+            preload: path.join(__dirname, './preload.js')
+        }
+    });
+
+    // Allow you to open devtools
+    globalShortcut.register('CommandOrControl+I', () => { mainWindow.webContents.openDevTools(); });
+
+    // if is dev mode passe argument to true
```
Comment (greg7859) [inline main.js]: mix french and english ;)
and the next lines are not working as describing by the comment

File: main.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
+const { app, BrowserWindow, globalShortcut, ipcMain} = require('electron');
+const path = require('path');
+const rootPath = require('electron-root-path').rootPath;
+const log = require('electron-log');
+require('./src/menu.js');
+const {getLogTransportConsole, getLogTransportFile, getLogResolvePath} = require('./src/lib/logFormat');
+const ipc = ipcMain;
```
Comment (greg7859) [inline main.js]: Why did you store ipcMain in the ipc variable?

Is it necessary?

File: src/server.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+
+window.addEventListener('DOMContentLoaded', () => {
+    // Method to format the writings logs
+    getLogTransportConsole();
+    getLogTransportFile();
+    getLogResolvePath();
+
+
+    console.log('Server started');
```
Comment (greg7859) [inline src/server.js]: for debug?

File: src/server.js [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+    app
+    .use(cors())
+    .use(bodyParser.json())
+    .use(express.static(path.join(__dirname, '../static')))
+    .use(express.static(path.join(__dirname, '../static/help/assets/style')))
```
Comment (greg7859) [inline src/server.js]: Is it mandatory to declare all sub folders?

File: src/utils/ConstructURL.js [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+
+    /**
+     * Return the server port number
+     * 
+     * @returns {number}
+     */
+    static getServerPort() {
+        if(!configViewer.server_port){
+            return 60000;
+        }else if(typeof configViewer.server_port === 'string'){
```
Comment (greg7859) [inline src/utils/ConstructURL.js]: Format your source code

File: src/server.js [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+        res.setHeader('Content-Type', 'application/json');
+
+        if(res.status === 500){
+            log.error(req, res);
+            res.send('Internal Server Error');
+        }
+
+        ipc.send('requested-url', req.body.url);
+
+        log.info(`POST request body ${JSON.stringify({url: req.body.url})} was sending succesfully`);
```
Comment (greg7859) [inline src/server.js]: I think the level should be debug

**Review by greg7859 — CHANGES_REQUESTED** [review]

File: src/server.js (hunk 5 above)
Comment (flo7842) [reply src/server.js]: yes because if i don't do that the css file is not recognized by the viewer.

**Pull request merged**